sections/about-hero/about-hero.module.scss

```diff
@@ -0,0 +1,90 @@
+@import "../../styles/styles.scss";
+@import "../../styles/spacing.scss";
+@import "../../styles/typography/typography.scss";
+
+.about-hero-container {
+  z-index: 1;
+}
+
+.about-hero {
+  z-index: 0;
+  padding: $space-xxlarge 0;
+  display: flex;
+  align-content: center;
+  align-items: center;
+  background: $primary;
+  // box-sizing: content-box !important;
+  position: relative;
+
+  @include md {
+    height: 400px;
+  }
+
+  &-container {
+    @include page-margin;
+
+    @include sm {
+      text-align: center;
+    }
+
+    @include lg {
+      max-width: 760px;
+    }
+
+    h1 {
+      font-weight: 600;
+      line-height: 42px;
+      font-size: 32px;
+      letter-spacing: -1px;
+      margin: 0 0 $base * 4;
+      color: black;
+
+      span {
+        display: inline-block;
+      }
+
+      @include md {
+        line-height: 70px;
+        font-size: 64px;
```

Comment:
let's also add here:
```
@include md {
	margin-bottom: $space-xxlarge;
}
```

That way, it looks a bit more balanced in the centre because of the stats container overly